src/style/partials/abstract/_mediaQuery.scss

```diff
@@ -0,0 +1,196 @@
+@use "../abstract/variables" as *;
+
+@media (max-width: 1010px) {
+    .contentPage{
+        .h2{
+            background-position: left;
+            height: 54vh;
+        }
+        form {
+            height: 30vh;
+            .plantInfo{
+                display: flex;
+                justify-content: center;
+                align-items: center;
+                margin-bottom: 2rem;
+            }
+            label{
+                font-size: 2.5rem;
+            }
+            #plantName, #wateringFreq{
+                margin: 0 0 0 .3rem;
+                width: 50%;
+            }
+            .add{
+                margin-top: 3rem;
+            }
+        }
+        .resultDisplay{
+            .remove{
+                margin-left: 1rem;
+            }
+        }
+    }
+}
+
+
+/* Portrait tablet and small desktops */
+@media (max-width: 945px) {
+    .contentPage{
+        form{
+            .plantName, .plantCare {
+                display: flex;
+                flex-direction: column;
+                align-items: center;
+                width: 50rem;
+               label{
+                 font-size: 2.2rem;
+                 margin-bottom: 1.2rem;
+               }
+               input, select, option{
+                font-size: 2rem;
+               };
+            }
+        }
+    }
+}
+
+/* Landscape phone to portrait tablet */
+@media (max-width: 810px) {
+    .homePage{
+        h1{
+            font-size: 8rem;
+        }
+    }
+    .contentPage{
+        h2{
+            font-size: 7rem;
+        }
+        form{
+            label{
+                font-size: 1.8rem;
+                padding-left: 1rem;
+                margin-right: 1rem;
+                }
+            input, select, option {
+                font-size: 1.6rem;
+            }
+        }
+        // Buttons
+        .changePage, .add, .remove, .complete, .backToHome{
+            width: 50%;
+            font-size: 1.6rem;
+            padding: 1rem 3rem;
+            text-align: center;
+        }
+
```

Comment:
in between ~400-800px, your buttons stack on your plants added, and they're also stacked... maybe want to have them centered so it looks a little cleaner